src/content/chainlink-automation/overview/getting-started.mdx

```diff
@@ -65,21 +65,21 @@ Click the tabs below to use Chainlink Automation with each type of trigger:
 
       1. Navigate to the [Chainlink Automation app](https://automation.chain.link/) and connect to **Arbitrum Sepolia** in the top dropdown menu.
 
-      2. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://automation.chain.link/arbitrum-sepolia).
+      1. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://automation.chain.link/arbitrum-sepolia).
 
-      3. Click **Register new Upkeep** and select **Custom logic** trigger.
+      1. Click **Register new Upkeep** and select **Custom logic** trigger.
 
-      4. Under _Target contract address_, enter `0xC3d3656868594db09abD410821c9F9ab9812B4d1`. This contract uses logic stored onchain and onchain state to determine when to perform, incrementing a counter. It is an example of an Automation-compatible contract and can be read [here](https://sepolia.arbiscan.io/address/0x6C0AAAeBcDb6F5D03759B8BF14b47BE491755530#readContract). Click **Next**.
+      1. Under _Target contract address_, enter <Address contractUrl="https://sepolia.arbiscan.io/address/0x6C0AAAeBcDb6F5D03759B8BF14b47BE491755530" />. This contract is an Automation-compatible contract that uses logic stored onchain and onchain state to determine when to increment a counter. View the source code [here](https://sepolia.arbiscan.io/address/0x6C0AAAeBcDb6F5D03759B8BF14b47BE491755530#readContract). Click **Next**.
 
          <ClickToZoom src="/images/automation/getting-started/gs_clu_contract_address.png" />
 
-      5. Enter an _Upkeep name_, your public key address under _Admin Address_, 500000 under _Gas limit_, and 0.1 under _Starting balance (LINK)_.
+      1. Enter an _Upkeep name_, your public key address under _Admin Address_, 500000 under _Gas limit_, and 0.1 under _Starting balance (LINK)_.
```

Comment:
Import `CopyText` then add copy/paste component like this:
`<CopyText text="500000" code/>`
`<CopyText text="0.1" code/>`